auslib/blobs/base.py

```diff
@@ -78,9 +78,17 @@ def merge_dicts(ancestor, left, right):
      * A type mismatch of unicode vs string is OK as long as the text is the same
        (Any other type mismatches result in a failure to merge.)
     """
+    # We can't use "logging" directly, because it ignores our custom code in log.py
+    log = logging.getLogger(__name__)
     result = {}
     dicts = (ancestor, left, right)
     for key in set(key for d in dicts for key in d.keys()):
+        # Extra logging information to help debug https://bugzilla.mozilla.org/show_bug.cgi?id=1501167
+        # This is a very large message, so we limit it as much as possible to reduce spam.
+        if key == "completes" and ancestor.get("appVersion") and "a1" not in ancestor["appVersion"]:
```

Comment:
Could this be further limited by testing for `key in left and key in right` ?

Reply:
I would think that it would need to be both in left and right for this to happen, but I'm also unclear about the STR :(